styles/badge.css

```diff
@@ -0,0 +1,59 @@
+@import url(/styles/docs.css);
+@import url(/styles/index.css);
+
+.icon-button {
+  position: relative;
+  width: 5rem;
+  height: 5rem;
+  color: var(--icon-color);
+  background: var(--icon-background-color);
+  border: none;
+  outline: none;
```

Comment:
I think, outline: none is not considered to be good practice..might look for an alternative, I can't remember at this moment

Reply:
Use 1px, transparent instead